### PR TITLE
bpo-38175: Fix a memory leak in comparison of sqlite3.Row objects.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue38175

### CODE REVIEW

File: Modules/_sqlite/row.c [hunk 1]
```diff
@@ -187,19 +187,22 @@ static Py_hash_t pysqlite_row_hash(pysqlite_Row *self)
     return PyObject_Hash(self->description) ^ PyObject_Hash(self->data);
 }
 
-static PyObject* pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other, int opid)
+static PyObject *
+pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other, int opid)
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: Nit: I prefer to not include cosmetic-only changes in such patch. Unfortunately, this isn't only case where sqlite3 codebase doesn't follow our general conventions, but I'd like to change them only when we need to change the definition `pysqlite_row_richcompare()`.

File: Misc/NEWS.d/next/Library/2019-09-15-10-30-33.bpo-38175.61XlUv.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix da memory leak in comparison of :class:`sqlite3.Row` objects.
```
Comment (berkerpeksag) [inline Misc/NEWS.d/next/Library/2019-09-15-10-30-33.bpo-38175.61XlUv.rst]: Typo: da -> a

**Review by berkerpeksag — COMMENTED** [review]
LGTM, the new code looks much better, thank you very much! I only have two minor comments on the patch.

**Review by berkerpeksag — APPROVED** [review]
LGTM, thank you!

**Comment by serhiy-storchaka** [comment]
Thank you for your review @berkerpeksag!

**Pull request merged**